extended/src/main/java/io/kubernetes/client/extended/leaderelection/LeaderElector.java

```diff
@@ -25,7 +60,7 @@
   private ExecutorService leaseWorkers = Executors.newSingleThreadExecutor();
   private ExecutorService hookWorkers = Executors.newSingleThreadExecutor();
 
-  public LeaderElector(LeaderElectionConfig config) {
+  protected LeaderElector(LeaderElectionConfig config) {
```

Comment:
we cant tighten that for backward-compatibility.

Reply:
+1